test/check-machines-create

```diff
@@ -1851,14 +1851,16 @@ vnc_password= "{vnc_passwd}"
         # Attach some interface
         m.execute("virsh attach-interface --persistent VmNotInstalled bridge virbr0")
 
-        # Change the os boot firmware configuration
-        b.wait_in_text("#vm-VmNotInstalled-firmware", "BIOS")
-        b.click("#vm-VmNotInstalled-firmware")
-        b.wait_visible(".pf-c-modal-box__body")
-        b.select_from_dropdown(".pf-c-modal-box__body select", "efi")
-        b.click("#firmware-dialog-apply")
-        b.wait_not_present(".pf-c-modal-box__body")
-        b.wait_in_text("#vm-VmNotInstalled-firmware", "UEFI")
+        # https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1006324
+        if "debian-stable" != m.image or m.execute("virsh --version") >= "8.1.0":
```

Comment:
I meant if you check for precise image names, you should drop the virsh --version comparison.